tomviz/modules/ModuleManager.cxx

```diff
@@ -190,6 +190,27 @@ bool ModuleManager::hasRunningOperators()
   return false;
 }
 
+QList<DataSource*> ModuleManager::dataSources()
+{
+  QList<DataSource*> ret;
+  std::copy(d->DataSources.begin(), d->DataSources.end(),
+            std::back_inserter(ret));
```

Comment:
Why the copy?

Reply:
The `d->DataSources` variable is a `QList<QPointer<DataSource>>`. I was copying them to convert them to a `QList<DataSource*>`. We could potentially also return a `QList<QPointer<DataSource>>`, but I thought most places would prefer the shorter variable name (plus, other functions in `ModuleManager` are returning QLists to raw pointers as well).